rpcs3/Input/ds4_pad_handler.cpp

```diff
@@ -949,7 +949,8 @@ void ds4_pad_handler::apply_pad_data(const std::shared_ptr<PadDevice>& device, c
 	if (!ds4_dev || !pad)
 		return;
 
-	auto profile = ds4_dev->config;
+	pad_config* ds4_config = ds4_dev->config;
+	auto profile = ds4_config;
```

Comment:
either use ds4_config or profile, but not both

Reply:
Rewrite it.
https://github.com/RPCS3/rpcs3/pull/9346/commits/808b3176ba21d2195d76ce0156480e673e99a008